compact/ecal/barrel_interlayers.xml

```diff
@@ -37,6 +37,26 @@
     <constant name="EcalBarrel_EpoxyThickness"       value="100*um"/>
     <constant name="EcalBarrel_CarbonThickness"      value="0.5*mm"/>
     <constant name="EcalBarrel_CarbonSpacerWidth"    value="4*mm"/>
+
+    <constant name="EcalBarrel_AstroPix_width"       value="2*cm"/>
+    <constant name="EcalBarrel_AstroPix_length"      value="2*cm"/>
+    <constant name="EcalBarrel_AstroPix_margin"      value="200*um"/>
+    <constant name="EcalBarrel_AstroPix_thickness"
+      value="EcalBarrel_SiliconThickness
+      + EcalBarrel_ElectronicsThickness
+      + EcalBarrel_CopperThickness
+      + EcalBarrel_KaptonThickness
+      + EcalBarrel_EpoxyThickness"/>
+
+    <constant name="EcalBarrel_Stave_width"          value="EcalBarrel_AstroPix_width + 2. * EcalBarrel_AstroPix_margin"/>
+    <constant name="EcalBarrel_Stave_length"         value="EcalBarrel_Calorimeter_length"/>
+    <constant name="EcalBarrel_StaveTilt_angle"      value="10*degree"/>
+    <constant name="EcalBarrel_Stave_ModuleRepeat"   value="floor(EcalBarrel_Calorimeter_length / (EcalBarrel_AstroPix_length + EcalBarrel_AstroPix_margin))"/>
+
+    <constant name="EcalBarrel_ImagingFirstLayerThickness"
+      value="EcalBarrel_AstroPix_thickness
+      + EcalBarrel_CarbonThickness"/>
```

Comment:
For now, let's make the first layer 1.5 cm since it needs to accommodate the fan blade rotation.
```suggestion
    <constant name="EcalBarrel_ImagingFirstLayerThickness" value="1.5*cm"/>
```

Reply:
For now, avoid distinguishing first layer entirely.